addons/mail/models/mail_activity.py

```diff
@@ -435,22 +439,34 @@ def write(self, values):
                 self.env[activity.res_model].browse(activity.res_id).message_subscribe(partner_ids=[activity.user_id.partner_id.id])
                 if activity.date_deadline <= fields.Date.today():
                     self.env['bus.bus'].sendone(
-                        (self._cr.dbname, 'res.partner', activity.user_id.partner_id.id),
-                        {'type': 'activity_updated', 'activity_created': True})
+                        activity.user_id.partner_id,
+                        'mail.activity_updated',
+                        {
+                            'activity_created': True,
+                        }
+                    )
             for activity in user_changes:
                 if activity.date_deadline <= fields.Date.today():
                     for partner in pre_responsibles:
                         self.env['bus.bus'].sendone(
-                            (self._cr.dbname, 'res.partner', partner.id),
-                            {'type': 'activity_updated', 'activity_deleted': True})
+                            partner,
+                            'mail.activity_updated',
+                            {
+                                'activity_deleted': True,
+                            }
+                        )
         return res
 
     def unlink(self):
         for activity in self:
             if activity.date_deadline <= fields.Date.today():
                 self.env['bus.bus'].sendone(
-                    (self._cr.dbname, 'res.partner', activity.user_id.partner_id.id),
-                    {'type': 'activity_updated', 'activity_deleted': True})
+                    activity.user_id.partner_id,
+                    'mail.activity_updated',
+                    {
+                        'activity_deleted': True,
+                    }
+                )
```

Comment:
guideline: single line
payload change: I think this key is useless, `activity_deleted` should be the type, and no payload.